### ISSUE TITLE
All timers disappeared.. 

### ISSUE THREAD

**Issue by Patrick1610** [ralfvd/nl.bevlogenheid.countdown#27]
All my timers are gone after an automatic update.... pleas help!

**Comment by ralfvd** [comment]
What Homey version are you using? And what Countdown version?  Do you know the latest version that it still worked for you?

**Comment by Patrick1610** [comment]
Countdown 2.0.3
Homey 1.5.13

I noticed Saturday that it didn't work anymore, and just noticed that the timers are gone. 
Is it possible to get them back? Or do I have to set it up manually?

**Comment by ralfvd** [comment]
They probably then disappeared during the update to 2.0.2 where an old bug was re-introduced. Although that should only cripple the flows, not the timers themselves.

Are the timers gone in the settings page?

Maybe you used the 'export' function recently, available in the settings page? If so, you could re-import your timers in the settings page.

This is the second mention of disappearing timers, so I'll continue investigation why this could be happening.

**Comment by Patrick1610** [comment]
Yes, the timers are gone in the settings page, but the flows don't show an error. 
In my flow the countdown card still shows set 'Keuken' to 120s for example, but the 'Keuken' timer does not exist. 

I don't have a backup made. 
When I re-add 'Keuken' now, does the flow work automatically? Or do I have to edit the flow also?

**Comment by ralfvd** [comment]
It should work automatically once you re-enter the timer in the settings
page.


On 10 December 2018 at 10:20:56, Patrick1610 (notifications@github.com)
wrote:

Yes, the timers are gone in the settings page, but the flows don't show an
error.
In my flow the countdown card still shows set 'Keuken' to 120s for example,
but the 'Keuken' timer does not exist.

I don't have a backup made.
When I re-add 'Keuken' now, does the flow work automatically? Or do I have
to edit the flow also?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/ralfvd/nl.bevlogenheid.countdown/issues/27#issuecomment-445745474>,
or mute the thread

**Comment by Patrick1610** [comment]
Okay, I'll get started with that ;) Thanks for thinking with me!

**Comment by WimHe** [comment]
I would like to report the same issue.
Timers are sudden gone a few days ago. I entered all the names in the setting page and it worked again.
Today Homey became unreachable from the app and all Hue lights and Nanoleaf were not getting updated (maybe due to the timer issue, i cannot be sure). Homey had to be rebooted. Then the times are all gone again. Now i have to re-enter them again. 
Is there any logging that might shed light on Homey becoming unrechable?
I hope i won't need to do this every few days.

**Comment by Keverjeroen** [comment]
Today all my timers disapeared too. Only thing I remember doing recently was rebooting Homey. I am using:
- CountDown v2.0.3
- Homey v1.5.13

I reentered the counters in the settingspage, and it looks like they are all working now.

**Comment by jhilhorst** [comment]
I just had the same experience today.
Lost all timers after a reboot of Homey

    CountDown v2.0.3
    Homey v1.5.13

**Comment by ralfvd** [comment]
Do you know if there was a countdown active during the reboot?

There is a known issue in the homey firmware 1.5 that saving settings (and each countdown timer is basically a setting) could make all settings disappear. 

This won’t be fixed by athom as they are gearing up for the 2.0 release. but I need to check why this is happening more often with the latest countdown version. 

In the meantime, protect yourself and export the countdown settings, so if it happens again, you can import the countdown timers very easily.

**Comment by jhilhorst** [comment]
There was no timer running. All timers are light related and i rebooted during the day.

**Comment by Keverjeroen** [comment]
I am not sure if there was a counter running, but there is a big change there was.
